src/utils/swapErrorToUserReadableMessage.tsx

```diff
@@ -12,11 +22,7 @@ export function swapErrorToUserReadableMessage(error: any): string {
     }
   }
 
-  let reason: string | undefined
```

Comment:
Can this be done in a separate PR? It expands the scope of this change.

Reply:
we need this change to actually detect user rejection. the change in this file is just a small refactor so i don't duplicate code.